base/install-functions.sh

```diff
@@ -15,6 +15,9 @@ install_module() {
 
     cd $module_name
     echo "$release_content" > configure/RELEASE
+    if [ -n "$NEEDS_TIRPC" ]; then
+        echo "TIRPC=YES" > configure/CONFIG_SITE.local
```

Comment:
I think we should better define how we are going to handle local files versus their counterpart.

For instance, this wouldn't have the desired effect when building pmac (#26). In this case, `CONFIG_SITE.local` will never be read, as [the upstream `CONFIG_SITE`][1] doesn't `include` the `.local` file (despite we currently overwriting the file completely).

Is that a single exception or it might be other modules that get rid of the auto-generated `include`s?

Nevertheless, I think we should append to the file instead, as it is the last step before building the module and other settings will eventually be required.

[1]: https://github.com/dls-controls/pmac/blob/2-6-1/configure/CONFIG_SITE